Removed executable bit

toin0u opened the pull request.

toin0u committed:
Removed executable bit

fabien-d commented:
this can no longer be merged automatically.

Could you also provide some info (or link to info) on this? Just for my benefit. I get removing the gist of it, but would like to get more background!

toin0u commented:
Sure I just rebased !

I just removed the [UNIX executable bit](http://en.wikipedia.org/wiki/Filesystem_permissions) from grunt.js and package.json files because they are obviously not executables. It's just done by `chmod a-x` to these files.
